Source/Panels/PatternBrowserPanel.cpp

```
@@ -352,10 +354,65 @@ void PatternBrowserPanel::renderContent(AestraUI::NUIRenderer& renderer) {
     
     if (isEmpty) {
-        std::string emptyText = (m_mode == BrowserMode::Patterns) 
-            ? "No patterns" 
-            : "No clips loaded\nDrop files here";
-        renderer.drawTextCentered(emptyText, listRect, 12.0f, m_textColor.withAlpha(0.5f));
+        auto& theme = AestraUI::NUIThemeManager::getInstance();
+        const bool dropActive = m_isDragOver;
+        const AestraUI::NUIRect stateCard{
+            bounds.x + 26.0f,
+            bounds.y + m_headerHeight + 34.0f,
+            bounds.width - 52.0f,
```

Cppcheck (2.20.0)

[style] The function 'set' is never used.

(unusedFunction)